api-spec/protobuf/tdex-daemon/v1/operator.proto

```diff
@@ -159,6 +159,14 @@ service OperatorService {
 
   // Returns info about volume and collected fees for specific time range
   rpc GetMarketReport(GetMarketReportRequest) returns(GetMarketReportResponse){}
+
+  // Returns the list of all available proto services
+  rpc ListProtoServices(ListProtoServicesRequest) returns(ListProtoServicesResponse){}
+}
+
+message ListProtoServicesRequest{}
+message ListProtoServicesResponse{
+  repeated string services = 1;
```

Comment:
this RPC must be moved to a dedicated proto into a `ReflectionService`.
It can't stay here because it wouldn't be possible to call the reflection API without knowing the version of the operator proto (v1/v2).